nemo_text_processing/text_normalization/en/taggers/math.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pynini
+from nemo_text_processing.text_normalization.en.graph_utils import (
+    NEMO_LOWER,
+    NEMO_SIGMA,
+    GraphFst,
+    delete_extra_space,
+    delete_space,
+    get_abs_path,
+)
+from nemo_text_processing.text_normalization.en.taggers.cardinal import CardinalFst
+from nemo_text_processing.text_normalization.en.taggers.decimal import DecimalFst
+from nemo_text_processing.text_normalization.en.taggers.fraction import FractionFst
+from pynini.lib import pynutil
+
+
+class MathFst(GraphFst):
+    """
+    Finite state transducer for classifying math
+
+
+    Args:
+        deterministic: if True will provide a single transduction option,
+            for False multiple transduction are generated (used for audio-based normalization)
+    """
+
+    def __init__(self, cardinal: CardinalFst, decimal: DecimalFst, fraction: FractionFst, deterministic: bool = True):
+        super().__init__(name="math", kind="classify", deterministic=deterministic)
+        delete_spaces = pynini.closure(pynutil.delete(" "), 1)
+        graph_var = pynutil.insert("name: \"") + NEMO_LOWER + pynutil.insert("\"")
```

Comment:
we actually have math here previously, could you either move this functionality there or remove functionality from there? so we don't have it at 2 places? https://github.com/NVIDIA/NeMo-text-processing/blob/main/nemo_text_processing/text_normalization/en/taggers/measure.py#L177